include/definitions.hpp

```diff
@@ -2,19 +2,13 @@
 #define DEFINITIONS_H
 
 #include "main.h"
+#include "drive.hpp"
 
 Controller master(CONTROLLER_MASTER);
 
-Motor rightA(19, MOTOR_GEAR_BLUE, false, MOTOR_ENCODER_DEGREES);
-Motor rightB(20, MOTOR_GEAR_BLUE, false, MOTOR_ENCODER_DEGREES);
-Motor_Group rightMotors({rightA, rightB});
+sdss_robotics_c::Drive drivetrain({15, -16, -17}, {-18, 19, 20}, E_MOTOR_GEAR_BLUE, E_MOTOR_BRAKE_COAST);
```

Comment:
Making it into a class is a good idea, especially considering that the structure of the drivetrain is unique